schemas/codex/coverage.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Coverage Schema",
+  "description": "Coverage Schema",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "beginCoverage": {
+      "type": "string",
```

Comment:
shouldn't we also set 'format' as 'date' for beginCoverage and endCoverage fields
(see https://github.com/joelittlejohn/jsonschema2pojo/wiki/Reference#format)
.. potentially with 'customPattern'

Reply:
Thanks for recommendation @dmtkachenko -- added both format and customPattern properties